### PR TITLE
Do not use htmlcache in development

### PR DESCRIPTION
**Pull request by gela-d**

### CODE REVIEW

**Comment by gela-d** [comment]
@tormozz48 @andrewblond @tadatuta @tavriaforever

**Comment by tadatuta** [comment]
@gela-d I'd suggest just to add env check right inside cache middleware

**Comment by gela-d** [comment]
@tadatuta I think it's not right way, because this verification does not needed in production.

**Comment by tadatuta** [comment]
@gela-d and you think `middleware__next` is much better for production then just one additional `if`? ;)

**Comment by gela-d** [comment]
@andrewblond @tormozz48 Boys, what u think?

**Comment by tormozz48** [comment]
Hello all. I think that environment condition check should be performed only once. 
We don't need to check it on every request

**Comment by tavriaforever** [comment]
up ?

**Comment by tavriaforever** [comment]
If nobody answer today, I will merge it tomorrow, because now I am having some problem with development.

File: src/blocks/server.blocks/middleware/middleware.node.js [hunk 1]
```diff
@@ -28,7 +30,7 @@ modules.define('middleware', [
                 slashes,
                 redirect,
                 router,
-                htmlCache,
+                config.get('NODE_ENV') !== 'development' ? htmlCache : next,
```
Comment (tadatuta) [inline src/blocks/server.blocks/middleware/middleware.node.js:33]: why not to add this check into `htmlCache` module and get rid of `middleware__next`?

**Comment by gela-d** [comment]
https://github.com/bem/bem-site-engine/pull/135

**Pull request closed without merging**